src/App.css

```diff
@@ -7,3 +7,10 @@
 .game-status-msg {
   margin-top: 0;
 }
+
+.game-change-settings-link{
+  font-size: 12px;
```

Comment:
I think that `14px` would look better. What do you think?

Reply:
I thinks `14px` feels intrusive and breaks the a bit the keyboard layout